### PR TITLE
Don't run coverage on pypy.

### PR DESCRIPTION
**Pull request by adiroiban**
# Scope

To speed up the tests we can run pypy tests without coverage.

I have checked the source code and I don't see any specific pypy handling in klein.

The coverage run on pypy is slow as there is no extension to speed the run.

# Changes

Don't run coverage on pypy.
Move coverage combine and XML generation into tox.ini ... we already had combine... but no xml generate.
XML generate in tox can help with a future usage of local `diff_cover`

# Before

See for example a run with coverage enabled on pypy

image

# After

With faster pypy and no coverage changes or decrease in coverage
  
image

### CODE REVIEW

**Comment by adiroiban** [comment]
This is ready for review.

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -253,17 +259,9 @@ jobs:
...
 
-      - name: "Combine coverage"
-        run: |
-          set -eux
-          pip install coverage[toml];
-          coverage combine;
-          coverage xml;
-        env:
-          COVERAGE_FILE: .tox/coverage
-
```
Comment (wsanchez) [inline .github/workflows/cicd.yml:264]: This is unrelated to the `pypy` change, right?

**Review by wsanchez — APPROVED** [review]

**Comment by wsanchez** [comment]
Yay zoom zoom

File: .github/workflows/cicd.yml (hunk 1 above)
Comment (adiroiban) [reply .github/workflows/cicd.yml:264]: Thanks for the review and the merge.

Yes. Not a hard requirements.

But this step was failing under pypy as there was no coverage.

Rather than add an `if` to this step, I found that tox already has combine and pip install logic, so I moved the XML part to tox.
